feat: Add admonition clarifying when security checks are run DOCS-610 (#1932)
codacy/docs: docs/getting-started/supported-languages-and-tools.md

    docs/repositories-configure/codacy-configuration-file.md (list of tool short names to use on the Codacy configuration file)
-->

!!! important

>     Codacy runs security and other analysis tools when code is pushed to your Git provider. These tools don't scan code for issues continuously.

Reviewer: ```suggestion
    Codacy runs security and other analysis tools when code changes are pushed to your repositories. These tools don't scan code for issues continuously.
```
